catalog/ui/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -25,11 +25,12 @@ define([
     'component/property/property',
     'component/dropdown/dropdown',
     'component/dropdown/dropdown.view',
+    './filter-param',    
```

Comment:
You just mean the consistency of absolute vs relative paths?
This could easily have been 'component/filter/filter-param', I guess the only difference is this view (filter-param) isn't intended to be used anywhere else.